prac_06/guitar.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+VINTAGE_YEAR = 50
+CURRENT_YEAR = date.today().year
+
+
+class Guitar:
+    """Represent a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance.
+
+        name: string, name of guitar object
+        year: int, date guitar was made
+        cost: float, cost of guitar
+        """
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return a string representation of a Guitar."""
+        return f"{self.name} ({self.year}) : ${self.cost:.2f}"
```

Comment:
Can also add ',' (:,.2f) to add commas in the price

Reply:
Good find! If you're still up I have fixed formatting and pushed changes.